frontend/src/layouts/styles/scrim.ts

```diff
@@ -0,0 +1,8 @@
+import { tw } from "@/utils/tailwind";
```

Comment:
Wasn't quite sure where to put these composed tailwind files, open to different folder structure @emma-sg

Reply:
I think these should go in `theme.stylesheet.css`? I don't see why these can't just be classes (probably in the `components` layer) with `@apply`

Reply:
That makes sense to me!

Reply:
Done--we should organize the stylesheet file into separate files at some point, looks like we would just need to enable the [import plugin](https://v3.tailwindcss.com/docs/adding-custom-styles#using-multiple-css-files).